src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/LicenseManager.cs

```diff
@@ -329,10 +321,11 @@ private static bool ValidateInternalRecursive(LicenseContext context, Type type,
                     if (license != null)
                     {
                         license.Dispose();
+#pragma warning disable IDE0059 // ValidateInternalRecursive does not null licence all the time
```

Comment:
Nit: Can we add a TODO in the comment with the url for the analyzer issue?

Reply:
Yep